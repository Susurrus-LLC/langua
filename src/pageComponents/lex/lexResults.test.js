import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import LexResults from './lexResults'

describe('LexResults', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<LexResults />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders accurately', () => {
    let tree = renderer.create(<LexResults />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
