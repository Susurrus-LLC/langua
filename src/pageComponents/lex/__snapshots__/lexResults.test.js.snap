// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LexResults renders accurately 1`] = `
<div
  className="results"
/>
`;
